app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+            }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {"errors": [format_linter_error(i) for i in errors],
+            "path": file_path,
+            "status": ("failed" if errors else "passed")}
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+
+    return [format_single_linter_file(i, linter_report[i])
+            for i in linter_report.keys()]
```

Comment:
Use .items() and rename i

Reply:
Is it necessary to use .items(), if i need only keys ?